actors/common/src/main/java/org/sunbird/learner/actors/coursebatch/CourseBatchManagementActor.java

```diff
@@ -862,18 +848,29 @@ private Date getDate(String key, SimpleDateFormat format, Map<String, Object> ma
   }
 
   private boolean isOrgValid(String orgId) {
-    Future<Map<String, Object>> respF =
-        esService.getDataByIdentifier(ProjectUtil.EsType.organisation.getTypeName(), orgId);
-    Map<String, Object> resp =
-        (Map<String, Object>) ElasticSearchHelper.getResponseFromFuture(respF);
-    if (resp != null && resp.size() > 0) {
-      ProjectLogger.log(
-          "CourseBatchManagementActor:isOrgValid: Organisation found in ES with id = " + orgId);
-      return true;
+    Response response;
+
+    boolean orgFound=false;
+    try {
+      response = userOrg.getOrganisationDetails(orgId);
+      Map<String,Object> orgDetails=(Map<String,Object>)new ObjectMapper().convertValue(response.get(RESPONSE), Map.class);
+      List<Map<String, Object>> list = (List<Map<String, Object>>) orgDetails.get(CONTENT);
```

Comment:
A better approach to test the validity of org is to verify input org id with the id in the first element of list in response.